### ISSUE TITLE
Train with another dataset

### ISSUE THREAD

**Issue by alikaz3mi** [JaidedAI/EasyOCR#495]
Hello, I know that you have said to ask questions about the dataset from https://github.com/clovaai/deep-text-recognition-benchmark repository. I have checked their repo. It is mentioned that create your own dataset, "put images in one folder, create their gt.txt". in gt.txt there is no sign of bounding boxes. I'd like to fine-tune both text detection and text recognizer. How I can do that? 

Thank you in advance.

**Comment by kimlia545** [comment]
@alikaz3mi 
"https://github.com/clovaai/deep-text-recognition-benchmark" repository is the place for the character recognition model.
Here for text detection.
"https://github.com/clovaai/CRAFT-pytorch" 
But the code for training is not included in this repository, and they cannot release the full training code for IP reason.

**Comment by saribolagh** [comment]
I want to fine-tune the https://github.com/JaidedAI/EasyOCR/releases/download/v1.3/english_g2.zip model but there are specific words in my data that I dont think there are in the this model. Which file should I modify? Also, do I need to convert the images and labels into .mdb, like in the repo  https://github.com/clovaai/deep-text-recognition-benchmark or it is not necessary ?

**Comment by alikaz3mi** [comment]
@kimlia545 
Thank you for your response.

**Comment by Manishsinghrajput98** [comment]
complete custom easy ocr recognition model training tutorial

https://www.youtube.com/watch?v=-j3TbyceShY

**Comment by aritra-deb** [comment]
> Hello, I know that you have said to ask questions about the dataset from https://github.com/clovaai/deep-text-recognition-benchmark repository. I have checked their repo. It is mentioned that create your own dataset, "put images in one folder, create their gt.txt". in gt.txt there is no sign of bounding boxes. I'd like to fine-tune both text detection and text recognizer. How I can do that?
> 
> Thank you in advance.

Hi, Did you find the solution?

**Comment by AliProgramer1386** [comment]
hey, this article is great [https://www.freecodecamp.org/news/how-to-fine-tune-easyocr-with-a-synthetic-dataset/](url)
if u follow each step carefully, it'll work.

**Comment by MengFoong** [comment]
@AliProgramer1386 Hi, May I know which article you follow to make the trained model to replace the normal model ?

**Comment by AliProgramer1386** [comment]
> [@AliProgramer1386](https://github.com/AliProgramer1386) Hi, May I know which article you follow to make the trained model to replace the normal model ?

@MengFoong
Hey, If you mean that how did I use the pretrained CRAFT and after that my trained network, I have to say I did not do that.
I just need my model for predicting the text of one word images. 
My custom data was something like this: 
[https://github.com/amirabbasasadi/Shotor](url)
